delta/app/src/main/resources/application-cassandra.conf

```diff
@@ -10,7 +10,7 @@ akka {
         keyspace-autocreate = ${app.database.cassandra.keyspace-autocreate}
         tables-autocreate = ${app.database.cassandra.tables-autocreate}
         keyspace = ${app.database.cassandra.keyspace}
-        pubsub-notification = on
+        pubsub-notification = off
```

Comment:
According to the doc:

```
# Setting pubsub-notification to "off" will disable the journal sending these announcements.
    # When enabled with `on` it will throttle the number of notfication messages per tag to at most 10 per second.
    # Alternatively a duration can be defined, such as pubsub-notification=300ms, which will be throttling interval
    # for sending the notifications.
```
It may better to set a duration there as 10 times per second is a little aggressive ?